libs/sdk-backend-spi/src/workspace/execution/index.ts

```diff
@@ -141,6 +142,15 @@ export interface IPreparedExecution {
      */
     withDimensions(...dim: Array<IDimension | DimensionGenerator>): IPreparedExecution;
 
+    /**
+     * Changes postProcessing of an IPreparedExecution.
+     *
+     * @param postProcessing - Configuration that should be done with the data after they are obtained from the server
+     *  and before they are passed to the user.
+     * @returns new execution with the updated postProcessing
+     */
+    withPostProcessing(postProcessing: IPostProcessing): IPreparedExecution;
```

Comment:
Can we change this to `withDateFormat`? I mean the added postProcessing is a good change, but looking at the API as the user (it is also apparent in the tests), wouldn't it be easier for the user to have a `withDateFormat` that would set the postProcessing accordingly?

Reply:
Thanks, I've updated the PR, please help me review again.